src/redux/sagas/utils/safeHelpers.ts

```diff
@@ -0,0 +1,402 @@
+import Web3 from 'web3';
+import { getTokenClient } from '@colony/colony-js';
+import { Contract, ethers } from 'ethers';
+import moveDecimal from 'move-decimal-point';
+import { AddressZero } from '@ethersproject/constants';
+import { FormatTypes } from 'ethers/lib/utils';
+
+import { Address, ModuleAddress, Safe, SafeTransactionData } from '~types';
+import { GNOSIS_AMB_BRIDGES, isDev, SAFE_NETWORKS } from '~constants';
+import { getArrayFromString } from '~utils/safes';
+
+import { erc721, ForeignAMB, HomeAMB, ZodiacBridgeModule } from './abis'; // Temporary
+
+export interface SafeTxData {
+  title: string;
+  transactions: SafeTransactionData[];
+  safeData: Omit<Safe, 'safeName' | 'moduleContractAddress'>;
+  annotationMessage?: string;
+}
+
+export interface SelectedSafe {
+  id: ModuleAddress; // Making explicit that this is the module address
+  profile: {
+    displayName: string;
+    walletAddress: Address; // And this is the safe address
+  };
+}
+
+export interface SelectedNFT extends SelectedSafe {
+  id: string; // id is address + id,
+}
+
+/* eslint-disable prefer-destructuring */
+const LOCAL_HOME_BRIDGE_ADDRESS =
+  process.env.LOCAL_HOME_BRIDGE_ADDRESS ||
+  // eslint-disable-next-line @typescript-eslint/no-var-requires, global-require, import/no-dynamic-require
+  require('../../../../amplify/mock-data/colonyNetworkArtifacts/safe-addresses.json')
+    .LOCAL_HOME_BRIDGE_ADDRESS;
+const LOCAL_FOREIGN_BRIDGE_ADDRESS =
+  process.env.LOCAL_FOREIGN_BRIDGE_ADDRESS ||
+  // eslint-disable-next-line @typescript-eslint/no-var-requires, global-require, import/no-dynamic-require
+  require('../../../../amplify/mock-data/colonyNetworkArtifacts/safe-addresses.json')
+    .LOCAL_FOREIGN_BRIDGE_ADDRESS;
+const LOCAL_ERC721_ADDRESS =
+  process.env.LOCAL_ERC721_ADDRESS ||
+  // eslint-disable-next-line @typescript-eslint/no-var-requires, global-require, import/no-dynamic-require
+  require('../../../../amplify/mock-data/colonyNetworkArtifacts/safe-addresses.json')
+    .LOCAL_ERC721_ADDRESS;
+const LOCAL_SAFE_ADDRESS =
+  process.env.LOCAL_SAFE_ADDRESS ||
+  // eslint-disable-next-line @typescript-eslint/no-var-requires, global-require, import/no-dynamic-require
+  require('../../../../amplify/mock-data/colonyNetworkArtifacts/safe-addresses.json')
+    .LOCAL_SAFE_ADDRESS;
+const LOCAL_SAFE_TOKEN_ADDRESS =
+  process.env.LOCAL_SAFE_TOKEN_ADDRESS ||
+  // eslint-disable-next-line @typescript-eslint/no-var-requires, global-require, import/no-dynamic-require
+  require('../../../../amplify/mock-data/colonyNetworkArtifacts/safe-addresses.json')
+    .LOCAL_SAFE_TOKEN_ADDRESS;
+/* eslint-enable prefer-destructuring */
```

Comment:
how about consolidating these eslint flags into the top level disable?
I tested this directive and it works:
```
/* eslint-disable prefer-destructuring, @typescript-eslint/no-var-requires, global-require, import/no-dynamic-require */
<imports>
/* eslint-enable prefer-destructuring, @typescript-eslint/no-var-requires, global-require, import/no-dynamic-require */
```